template/src/core/app/appStateReducer.ts

```diff
@@ -0,0 +1,30 @@
+const initialAppState = {
+  account: null as null | Record<string, any>
+};
+
+export type AppStateReducerAction = {
+  type: "SET_HAS_LOGGED_IN";
+  account: null | Record<string, any>;
+};
+
+function appStateReducer(state: typeof initialAppState, action: AppStateReducerAction) {
+  let newState = state;
+
+  switch (action.type) {
+    case "SET_HAS_LOGGED_IN": {
+
+      newState = {
+        ...state,
+        account: action.account
+      };
+      break;
+    }
```

Comment:
```suggestion
    case "SET_LOGGED_IN_ACCOUNT": {
      newState = {
        ...state,
        account: action.account
      };
      break;
    }
```
- let's change the action name as well.
- there's no need for a blank line here